client/my-sites/site-settings/controller.js

```diff
@@ -111,38 +99,29 @@ module.exports = {
 	importSite( context ) {
 		renderPage(
 			context,
-			<SiteSettingsComponent sites={ sites } section="import" />
+			<SiteSettingsComponent section="import" />
 		);
 	},
 
 	exportSite( context ) {
 		renderPage(
 			context,
-			<SiteSettingsComponent sites={ sites } section="export" />
+			<SiteSettingsComponent section="export" />
 		);
 	},
 
 	guidedTransfer( context ) {
 		renderPage(
 			context,
-			<SiteSettingsComponent sites={ sites } section="guidedTransfer" hostSlug={ context.params.host_slug } />
+			<SiteSettingsComponent section="guidedTransfer" hostSlug={ context.params.host_slug } />
 		);
 	},
 
 	deleteSite( context ) {
-		let site = sites.getSelectedSite();
```

Comment:
I've removed everything related to the deleteSite route from this PR as it was here previously but belonged to another branch.